src/cgi/sfs.cgi.in

```diff
@@ -1652,7 +1652,8 @@ if "HD" in sectionset:
                             wtime = usecwritemax
                             fsynctime = usecfsyncmax
                         if HDorder == 1 or HDorder == 0:
-                            sf = (ip1, ip2, ip3, ip4, port, data[1:plen + 1])
+                            sf = (ip1, ip2, ip3, ip4, port, entry[1:plen + 1]
+                                  .decode('utf-8'))
```

Comment:
Is there a test checking for this to be correctly working?

Reply:
Even though there is a test for entire cgi, this issue is not reported and was accidentally detected.

Before modification:
![image](https://github.com/user-attachments/assets/7c1c3e4c-c6af-4920-8283-527afb397b0b)

After modification:
![image](https://github.com/user-attachments/assets/66f8a83d-9271-4525-beb2-f7e9410d9720)

Reply:
Ok perfect